nubis/puppet/sites.pp

```diff
@@ -30,6 +30,14 @@
   override      => ['All'],
 }
 
+nubis::static { 'planet-bugzilla':
+  servername    => 'planet.bugzilla.org',
+  serveraliases => [
+    'planet-bugzilla.allizom.org',
+  ],
+  override      => ['All'],
```

Comment:
Is that actually needed? Otherwise, it opens us up to errors/failures introduced via .htaccess files in the source repository.

I’d try and avoid enabling overrides unless actually necessary, and even then, keep them limited.

Reply:
Yeah, good catch. I don't think I need to control which directives are placed in a .htaccess file. I'm dropping that line from the configuration.